components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -373,8 +397,7 @@ func (m *Motor) readReg(ctx context.Context, addr uint8) (int32, error) {
 	value <<= 8
 	value |= int32(rbuf[4])
 
-	// m.logger.Debug("ReadR: ", rbuf)
-	// m.logger.Debug("Read: ", value)
+	m.logger.Debugf("Read from 0x%x: %d (%v)", addr, value, rbuf[1:])
```

Comment:
By default, debug data is not actually logged, so this shouldn't get in the way. but if you turn on debug info, you probably want this. So, I uncommented it. I'm willing to comment it back out if you want, though.

Reply:
Example log statements:
```
Write to 0x6c: [0 1 0 195]
Read from 0x35: 1792 ([0 0 7 0])
```